source/uwp/Renderer/dll/CppWinRTIncludes.h

```diff
@@ -10,22 +10,31 @@
 #include <winrt/Windows.Web.Http.h>
 #include <winrt/Windows.Web.Http.Filters.h>
 #include <winrt/Windows.UI.Core.h>
-#include <winrt/Windows.UI.Xaml.Automation.h>
-#include <winrt/Windows.UI.Xaml.Automation.Peers.h>
-#include <winrt/Windows.UI.Xaml.Controls.h>
-#include <winrt/Windows.UI.Xaml.Controls.Primitives.h>
-#include <winrt/Windows.UI.Xaml.Input.h>
-#include <winrt/Windows.UI.Xaml.Markup.h>
-#include <winrt/Windows.UI.Xaml.Media.h>
-#include <winrt/Windows.UI.Xaml.Shapes.h>
-#include <winrt/Windows.UI.Xaml.Documents.h>
-#include <winrt/Windows.UI.Xaml.Media.Imaging.h>
 #include <winrt/Windows.Data.Json.h>
 #include <winrt/Windows.Foundation.h>
 #include <winrt/Windows.Foundation.Collections.h>
 #include <winrt/Windows.Data.Xml.Dom.h>
 #include <winrt/AdaptiveCards.ObjectModel.Uwp.h>
+
 #include <winrt/Windows.UI.Text.h>
+#include <winrt/Windows.UI.Xaml.Automation.h>
```

Comment:
Is there reason you moved #include <winrt/Windows.UI.Xaml.Automation.h> and others from line 12 to line 20?

Reply:
Yes! It's just not relevant (yet) haha. All the changes in this file are intended to make my life easier when ingesting WinUI 3. Depending on how big of a change that is, that may get lumped into this PR or a separate one that targets this one (and takes advantage of these changes.)